### ISSUE TITLE
Not able to reproduce the results

### ISSUE THREAD

**Issue by akhila1729** [FT-ZHOU-ZZZ/CMTA#4]
I tried to reproduce the results. But the model is overfitting. Same code and same train and val set.

**Comment by FT-ZHOU-ZZZ** [comment]
Could you provide more details please?
You want to reproduce same results reported in paper? Or other problem?

**Comment by akhila1729** [comment]
Yes. I tried reproducing the results. But observed that the validation loss is not falling. And I am getting Cindex as mentioned in the paper in one of the epochs. But while I tried to do prediction on a test dataset on which the model is not trained, I got cindex as 0.5 which is random result.

**Comment by FT-ZHOU-ZZZ** [comment]
Which TCGA dataset you are working on?
According to the limited information you provided , I am not sure what problem you have encountered.
BTW, if you want to exectly reproduce the same results reported in paper, you need to make sure all environments are consistent, including hardware devices, CUDA version, pytorch version, pathology features, etc.
Perhaps, there are some problems with your data processing.
It's better if you can provide more implementation details, such as dataset, log, output, etc.

**Comment by akhila1729** [comment]
Dataset - TCGA BRCA
Data processing using CLAM as mentioned in the repo
